packages/core/entity-list/src/components/Table/SelectRowNums.js

```diff
@@ -0,0 +1,44 @@
+import PropTypes from 'prop-types'
+import {useState} from 'react'
+import {EditableValue, Button} from 'tocco-ui'
+
+import {StyledButtonWrapper, StyledEditableValueWrapper} from './StyledComponents'
+
+const SelectNumRows = ({onOk, numOfRows}) => {
+  const options = [
+    {key: 1, display: '25'},
```

Comment:
suggestion(blocking): Use number value as key and use `key` for your value instead `display`

```
const options = [{key: 25: display: '25'}, ...]
const matchingValue = options.find(o => o.key === numOfRows)
...
onClick={() => onOk(value?.key)}
```

Then we could maybe remove all the type conversions expect the one when loading the preferences initially.